### PR TITLE
Default to 18.04 SDS release

### PR DESCRIPTION
**Pull request by vincent-legoll**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 openio_repository_products:
   sds:
-    release: '17.04'
+    release: '18.04'
```
Comment (cdelgehier) [inline defaults/main.yml:6]: every 6 months, this default will change so?

**Review by cdelgehier — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (vincent-legoll) [reply defaults/main.yml:6]: Do you want a `current` symlink to the latest and put this in the default value ?

Comment (racciari) [reply defaults/main.yml:6]: This value is relative to the branch of this project. Until we start to use release branch, let's change it every 6 months.

**Comment by racciari** [comment]
It seems that we need to modify the tests accordingly.

**Comment by vincent-legoll** [comment]
Argh, forgot about the tests...

**Pull request closed without merging**